hm_pyhelper/miner_json_rpc/client.py

```diff
@@ -1,7 +1,14 @@
 import requests
+import base64
+import base58
+
+import hm_pyhelper.protos.blockchain_txn_pb2 as blockchain_txn_pb2
```

Comment:
If name is the same can also be written more concisely as:

```python
from hm_pyhelper.protos import blockchain_txn_pb2
```

`as` is usually used when we want to rename the imported module in the context of current module.